packages/flutter/test/material/popup_menu_theme_test.dart

```diff
@@ -78,16 +82,21 @@ void main() {
       home: Material(
         child: Column(
           children: <Widget>[
-            PopupMenuButton<void>(
-              key: popupButtonKey,
-              itemBuilder: (BuildContext context) {
-                return <PopupMenuEntry<void>>[
-                  PopupMenuItem<void>(
-                    key: popupItemKey,
-                    child: const Text('Example'),
-                  ),
-                ];
-              },
+            Padding(
+              // The padding makes sure the menu as enough space to around to
```

Comment:
```suggestion
              // The padding makes sure the menu has enough space around it to
```

Reply:
Thank you, I created a new PR to fix this https://github.com/flutter/flutter/pull/111060